Update active-directory-saas-floqast-tutorial.md

naimoor opened the pull request:
Please also update the images tutorial_floqast_url.png and tutorial_floqast_url1.png as instructed per email.

azurecla commented:
Hi __@BrucePerlerMS__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

naimoor committed:
Update active-directory-saas-floqast-tutorial.md

Please also update the images tutorial_floqast_url.png and tutorial_floqast_url1.png as instructed per email.
